lib/common/options.c

```diff
@@ -663,8 +664,16 @@ pcmk__format_option_metadata(const char *name, const char *desc_short,
             }
         }
 
-        g_string_append_printf(s, "    <parameter name=\"%s\">\n",
+	if (flag)
+	{
+        	g_string_append_printf(s, "    <parameter name=\"%s\" unique=\"0\" >\n",
                                option_list[lpc].name);
+	}
+	else 
+	{
+        	g_string_append_printf(s, "    <parameter name=\"%s\">\n",
+                               option_list[lpc].name);
+	}
```

Comment:
Thanks for trying to keep the output the same, but the "unique" parameter was recently deprecated, so we don't need to show it here. You can revert this part and the new function argument.